backend/core/forms.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+
+
+class UserRegisterForm(UserCreationForm):
+    email = forms.EmailField()
+
+    class Meta:
+        model = User
+        fields = ["username", "email", "password1", "password2"]
```

Comment:
Those variable names aren't explicit enough plus they use numbers which I personally think should be avoided.  
password1 is "password", password2 is "confirmPassword"